openassessment/data.py

```diff
@@ -333,4 +337,135 @@ def _use_read_replica(self, queryset):
             queryset.using("read_replica")
             if "read_replica" in settings.DATABASES
             else queryset
-        )
\ No newline at end of file
+        )
+
+
+class AggregateOraData(object):
+    """
+    Aggregate all the ORA data into a single table.
+
+    Only works on MySQL databases.
+    """
+
+    @classmethod
+    def _read_replica_cursor_if_available(cls):
```

Comment:
Maybe call this `_read_replica_or_regular_cursor`?  The name as-is is slightly vague about what it returns in the case it is not available.